third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -1025,6 +1056,8 @@ func resourceSqlDatabaseInstanceUpdate(d *schema.ResourceData, meta interface{})
 		return err
 	}
 
+	// TODO if d.HasChange("settings") doesn't work, so we just send an empty request if only restoring from backup
```

Comment:
Thanks to https://github.com/hashicorp/terraform-plugin-sdk/issues/98 and `settings.authorized_networks`, d.HasChange("settings") always returns true.

As it stands the resource will send empty updates which causes an extra harmless request.